runtime/tutorials/chat_app.md

```diff
@@ -163,73 +140,154 @@ function broadcast_usernames() {
     }),
   );
 }
+```
+
+The `broadcast` function sends a message to all connected clients and the
+`broadcast_usernames` function sends the list of currently connected users to
+all clients.
+
+## Building the Frontend
+
+We'll build a simple UI that shows a text input and a send button and displays
+the sent messages, alongside a list of users in the chat.
+
+### HTML
+
+In your new project directory, create a `public` folder and add an `index.html`
+file and add the following code:
+
+```html
+<html>
+    <head>
+        <title>Deno Chat App</title>
+        <link rel="stylesheet" href="/public/style.css">
+        <script src="/public/app.js"></script>
+    </head>
+    <body>
+        <header>
+            <h1>🦕 Deno Chat App</h1>
+        </header>
+        <aside>
+            <h2>Users online</h2>
+            <ul id="users"></ul>
+        </aside>
+        <main>
+            <div id="conversation"></div>
+            <form>
+                <textarea id="data" placeholder="send message"></textarea>
+                <button type="submit" id="send">Send ᯓ✉︎</button>
+            </form>
+        </main>
```

Comment:
Let's move the script tag to just before the closing `</body>` tag. That way we don't need to explain the `load` event. We should also use `type="module"` for strict mode and stuff.

```suggestion
    </head>
    <body>
        <header>
            <h1>🦕 Deno Chat App</h1>
        </header>
        <aside>
            <h2>Users online</h2>
            <ul id="users"></ul>
        </aside>
        <main>
            <div id="conversation"></div>
            <form>
                <textarea id="data" placeholder="send message"></textarea>
                <button type="submit" id="send">Send ᯓ✉︎</button>
            </form>
        </main>
        <script src="/public/app.js" type="module"></script>
```